airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -549,7 +549,7 @@ def cast_property_type(self, property_name: str, column_name: str, jinja_column:
                 return f"try_parse({replace_operation} as {sql_type}) as {column_name}"
             if self.destination_type == DestinationType.CLICKHOUSE:
                 sql_type = jinja_call("type_date()")
-                return f"parseDateTimeBestEffortOrNull(trim(BOTH '\"' from {replace_operation})) as {column_name}"
+                return f"toDate(parseDateTimeBestEffortOrNull(trim(BOTH '\"' from {replace_operation}))) as {column_name}"
```

Comment:
Was returning `toDate(...)` helpful to convert `date_time` back to `date` in your case?

Reply:
Do you have local tests passed?

Reply:
@bazarnov yes, it helped.